src/Priestley.js

```diff
@@ -27,10 +27,14 @@ export default class Priestley extends Viz {
   constructor() {
 
     super();
+
     this._axis = new Axis().align("end").orient("bottom");
     this._axisConfig = {scale: "time"};
     this._axisTest = new Axis().align("end").gridSize(0).orient("bottom");
     this.end("end");
+    this._shapeConfig = assign({}, this._shapeConfig, {
+      ariaLabel: (d, i) => `${this._drawLabel(d, i)}, ${d.start} - ${d.end}.`
```

Comment:
We cannot reference `d.start` and `d.end`, because the user may change the associated accessors to reference a different key inside of their data. This should be:
```jsx
ariaLabel: (d, i) => `${this._drawLabel(d, i)}, ${this._start(d, i)} - ${this._end(d, i)}.`
```